.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
             - ubuntu-toolchain-r-test
           packages:
             - g++-5
+            - libboost-all-dev
```

Comment:
It should be the version present in Ubuntu Trusty(as configured by the dist directive in line 4) repositories, which is 1.54.0.1